src/DoctrineMiddleware.php

```diff
@@ -113,6 +113,7 @@ public function setup()
         if (!$this->setupMetadataDriver($config)) {
             throw new \RuntimeException('No Metadata Driver defined');
         }
+        $config->setAutoGenerateProxyClasses($this->getOption('auto_generate_proxies', true) ? true : false);
```

Comment:
When you specify in the documentation:
`auto_generate_proxies` bool indicating whether Doctrine should auto-generate missing proxies (default: true)

I'd really expect the value to be a boolean and not rely in the dynamic nature of PHP, put another way:

```php
$config = [
    'doctrine' => [
        'connection' => [],
        'auto_generate_proxies': 'I do not want this activated',
    ],
];
```

The string is cast to `true`, that's awkward

Replacing triple-equals to double would accomplish what you want without ternary operator

```php
$config->setAutoGenerateProxyClasses($this->getOption('auto_generate_proxies', true) === true);
```